test/systemtests/network_test.go

```diff
@@ -11,51 +11,52 @@ import (
 	. "gopkg.in/check.v1"
 )
 
-func (s *systemtestSuite) TestInfraNetworkAddDeleteVXLAN(c *C) {
+func (s *systemtestSuite) TestInfraNetworkAddDelete(c *C) {
 	if s.basicInfo.Scheduler == "k8" {
 		return
 	}
-	s.testInfraNetworkAddDelete(c, "vxlan")
-}
 
-func (s *systemtestSuite) TestInfraNetworkAddDeleteVLAN(c *C) {
-	if s.basicInfo.Scheduler == "k8" {
-		return
+	doneChan := make(chan struct{}, 2)
+
+	go s.testInfraNetworkAddDelete(c, EncapVXLAN, doneChan, 0)
+	go s.testInfraNetworkAddDelete(c, EncapVLAN, doneChan, 1)
+
+	for i := 0; i < 2; i++ {
+		<-doneChan
 	}
-	s.testInfraNetworkAddDelete(c, "vlan")
 }
 
-func (s *systemtestSuite) testInfraNetworkAddDelete(c *C, encap string) {
+func (s *systemtestSuite) testInfraNetworkAddDelete(c *C, encap string, doneChan chan struct{}, seq int) {
 
 	if s.fwdMode == "routing" && encap == "vlan" {
 		s.SetupBgp(c, false)
 		s.CheckBgpConnection(c)
 	}
 
-	for i := 0; i < s.basicInfo.Iterations; i++ {
+	defer func() { doneChan <- struct{}{} }()
+
+	tenantName := fmt.Sprintf("tenant-%d", seq)
+	tenant := &client.Tenant{TenantName: tenantName}
+	c.Assert(s.cli.TenantPost(tenant), IsNil)
+
+	for i := 0; i < 1; i++ {
 		var (
 			netNames = []string{}
 		)
 
 		numInfraNw := 3
 		for networkNum := 0; networkNum < numInfraNw; networkNum++ {
 			network := &client.Network{
-				TenantName:  "default",
+				TenantName:  tenantName,
```

Comment:
we need specific tests to be run under default tenant as well. We create that tenant as part of installation and We need to make sure that default tenant is also working. Thats the reason default was hard coded here.